.github/ISSUE_TEMPLATE/ask_a_question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Ask a question
+about: Ask a question for FullStory or the community
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Your Questions**
+A clear and concise description of your question and what you are trying to achieve.
+
+**Context**
+A broader context in which you’re asking the question, please provide any context that might be helpful for the recipient to know? This may include your use case, or a description of an unexpected behavior.
+
+**Attempts**
+Any attempts you've made to try to answer the question.
+
+**Additional comment**
+Add any other comment about the issue, or the reason and use case for this.
```

Comment:
```suggestion
Add any other comments about the issue.
```

Reply:
fixed 👍

Reply:
I'm not seeing this change reflected in the latest version.

Reply:
that's weird im seeing it here in the commit 🤔 

https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/pull/162/commits/a00eb297a5f361918c4d7f18628d8af7007b93e1